@here/harp-mapview-decoder/lib/TileDataSource.ts

```diff
@@ -246,6 +246,11 @@ export class TileDataSource<TileType extends Tile> extends DataSource {
         }
 
         tileLoader.loadAndDecode().then(() => {
+            // if decodedTile once set on tile - do nothing
+            if (tile.decodedTile !== undefined) {
```

Comment:
I tried to prevent toggling of m_forceHasGeometry when 
tile.forceHasGeometry(tileLoader.decodedTile === undefined || !tile.hasGeometry));
from 
https://github.com/heremaps/harp.gl/pull/306